### ISSUE TITLE
Deployment

### ISSUE THREAD

**Issue by code0wl** [ponzu-cms/ponzu#229]
ponzu is an awesome project, my deepest thanks to Boss Sauce Creative.
I appreciate maintainers' work and will be glad to help as I can.

Sadly, I am stuck with my deployment pipeline. Getting ponzu to build on the server is a breeze, but having issues connecting or loading in the content for the CMS when using the run --bind flag.

This seems to be only happening on the Debian platform and running node server enviroment for the frontend and ponzu instance outside of a docker container instance. 

Not sure if I am doing it correctly

**Comment by nilslice** [comment]
@code0wl - just making sure you intended to close this. You found a solution? 

I typically deploy the same way -- single Debian instance, no Docker. So I'd be happy to help if you still need it. 

Thanks for the kind words and welcome to Ponzu! :)

**Comment by code0wl** [comment]
Hey @nilslice 

Yeah solved, bit of a silly one really. I attempted to install Ponzu via go so I can get the CLI tool. So when building and running the project, it was not building the current Ponzu project I have worked on, but the obtained by running go get -g this-reposititory. I believe this one comes out of the box is a docs.go.

No content was showing because it was just starting the wrong instance.  Oopsy.
